pkg/db/matviews.go

```diff
@@ -3,29 +3,34 @@ package db
 import (
 	"fmt"
 	"strings"
+	"time"
 
 	"gorm.io/gorm"
 )
 
+const ReplaceTimeNow = "|||TIMENOW|||"
+const TimestampFormat = "2006-01-02 15:04:05"
```

Comment:
Are these only used here? They should start with a lower case character, otherwise they're exported

Reply:
yep, thanks!